src/index.ts

```diff
@@ -62,8 +62,7 @@ class LiskMigrator extends Command {
 		output: flagsParser.string({
 			char: 'o',
 			required: false,
-			description:
-				"File path to write the genesis block. If not provided, it will default to cwd/output/{v3_networkIdentifier}/genesis_block.blob. Do not use any value starting with the default data path reserved for Lisk Core: '~/.lisk/lisk-core'.",
+			description: `File path to write the genesis block. If not provided, it will default to cwd/output/{v3_networkIdentifier}/genesis_block.blob. Do not use any value starting with the default data path reserved for Lisk Core: '${DEFAULT_LISK_CORE_PATH}'.`,
```

Comment:
do we have v3_networkIdentifier? `networkIdentifier` was in general removed in favour of chainID?